routes/order.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+// Import modules
+const express = require('express');
+const Joi = require('joi');
+
+const validator = require('@byba/express-validator');
+const { checkAdmin } = require('../middleware');
+
+const {
+	helpers: {
+		SortHelper
+	}
+} = require(global.__lib);
+
+const router = express.Router();
+
+const idParamSchema = Joi.object({
+	id: Joi.string()
+		.guid()
+		.required()
+});
+
+router.get('/', checkAdmin, validator.query(Joi.object({
+	status: Joi.string().optional()
+})), async (req, res, next) => {
+	const orderWhere = {};
```

Comment:
Is it any worth this showing all orders for admin users, or *just my orders* for non-admin users?

Reply:
Again, linked the the idea of order attribution to individuals/organisations